.github/workflows/build_and_test.yml

```
@@ -11482,7 +11482,7 @@ jobs:
         mkdir tests\cmake\build
         cd tests\cmake\build
         cmake ..
```

It has to be done on the configuring cmake command.

```suggestion
        cmake .. -DCMAKE_CXX_STANDARD=20
```